### PR TITLE
UI replace muimaterial

### PR DESCRIPTION
**Pull request by Brianna-Flo**
## Description

<what this pull request is doing>

- Replace all instances of MUI material into MUI Joy components to allow the global css context to apply to all components
- apply global properties to components
- apply a color scheme

<what will be done in later PRs and not included here>

- solidifying the color scheme
- applying further global style properties to components

## Milestones
<the milestones or stories/features that this works towards>

- frontend design, migrating to react components

## Resources
<links to tutorials, code snippets, inspirations>
<if you took or translated specific code from the internet, please call it out here!>
https://mui.com/joy-ui/customization/approaches/#theming
https://mui.com/joy-ui/customization/themed-components/
https://mui.com/joy-ui/customization/theme-builder/

## Test Plan
<insert images or gifs of feature>
<otherwise, say how code was tested if not UI facing>
<any edge cases you might have specifically tested>

### CODE REVIEW

File: frontend/src/components/IngredientModal.tsx [hunk 1]
```diff
@@ -147,7 +152,7 @@ const IngredientModal: React.FC<GPIngredientModalProps> = ({
               />
             </FormControl>
           )}
-          <Box sx={{display: "flex", justifyContent: "space-between"}}>
+          <Box sx={{ display: "flex", justifyContent: "space-between" }}>
```
Comment (nithyad) [inline frontend/src/components/IngredientModal.tsx:155]: nit looks like theres a space after "space-between"

**Review by nithyad — APPROVED** [review]
